### PR TITLE
Fixed bug: Draw new label/tag by shortcut after annotate a tag blocks "Rectangle" selection button in side-bar

### LINKED ISSUE #4503: Draw new label/tag by shortcut after annotate a tag blocks "Rectangle" selection button in side-bar
### My actions before raising this issue
- [x] Read/searched [the docs](https://github.com/opencv/cvat/tree/master#documentation)
- [x] Searched [past issues](/issues)
- [x] Searched [Gitter](https://gitter.im/opencv-cvat/public)

We are using the "Standard" Workspace and a handful of labels and one tag with attributes for our annotation jobs. We often need to switch between placing a tag and placing a label.

### Expected Behaviour
1. Create a tag by clicking the tag symbol on the side-bar -> Tag gets created
2. Create a new tag by shortcut "n" or select by mouse the rectangle shape in the side-bar to create a new rectangle label

### Current Behaviour
1. Create a tag by clicking the tag symbol on the side-bar -> Tag gets created
2. Create a new tag by shortcut "n" -> Tag gets created
3. Pressing "n" again does nothing and the rectangle button is "locked" and cannot be clicked anymore, see image. 
<a href="https://imgbb.com/">bug-ui</a>

### Possible Solution
Current workaround is to select by mouse another label/tag from the side-bar, place it, delete it again and create the original label.

### Steps to Reproduce (for bugs)
1. Create a tag by clicking the tag symbol on the side-bar
2. Create a new tag by shortcut "n" 
3. Rectangle button is locked

No errors or warnings gets printed.

### Context
Same problem with [cvat.org](cvat.org), CVAT v2.0.0-alpha and v2.0.0 release version.

### Your Environment
- Server version: 2.0
- Core version: 4.2.0
- Canvas version: 2.13.1
- UI version: 1.35.0
- Docker version 20.10.12, build e91ed57
- Ubuntu 20.04

### PR DESCRIPTION
**Pull request by klakhov**
### Motivation and context

Resolved #4503 

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [x] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2022 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by bsekachev — DISMISSED** [review]

File: cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx [hunk 1]
```diff
@@ -32,7 +32,11 @@ function mapDispatchToProps(dispatch: any): DispatchToProps {
             dispatch(createAnnotationsAsync(sessionInstance, frame, states));
         },
         onRememberObject(labelID: number): void {
-            dispatch(rememberObject({ activeObjectType: ObjectType.TAG, activeLabelID: labelID }));
+            dispatch(rememberObject({
+                activeObjectType: ObjectType.TAG,
+                activeLabelID: labelID,
+                activeShapeType: undefined,
```
Comment (bsekachev) [inline cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx:38]: Hm, if we do not pass activeShapeType it anyway has ``undefined`` value, right? So, what has changed?

Comment (klakhov) [reply cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx:38]: It seems that specifying `undefined` changes overwrite behavior here - https://github.com/openvinotoolkit/cvat/blob/develop/cvat-ui/src/reducers/annotation-reducer.ts#L499

If `undefined` is default(aka property is not specified), old activeShapeType from state.drawing remains the same, and if it's specified, activeShapeType becomes overwritten. 
(I've tried to change changing spreading of payload and old state but had no luck)

Comment (bsekachev) [reply cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx:38]: Ok, interesting spread effect

Comment (bsekachev) [reply cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx:38]: But one more question here, ``activeControl `` variable is computed before this spread operator, basing only on payload. So, still not clear.

Comment (bsekachev) [reply cvat-ui/src/containers/annotation-page/standard-workspace/controls-side-bar/setup-tag-popover.tsx:38]: and it sets up as ``tag``.  Okay, I will debug it a bit

**Review by bsekachev — APPROVED** [review]

**Pull request merged**